js/apps/events-page/dataServices/metaService.js

```diff
@@ -0,0 +1,24 @@
+((app) => {
+	'use strict';
+
+	const metaService = ($http, $q) => {
+		const request = (endpointUrl) => {
+			return $q((resolve, reject) => {
+				$http.get(endpointUrl)
+					.then((response) => {
+						resolve(response.data);
+					}, (err) => {
```

Comment:
I think this can just be 
```javascript
    resolve(response.data); 
}, reject);
```

Reply:
I'm going to leave it as-is so I'm staged for better error handling in the future.